js/libs/quantimodo-api.js

```diff
@@ -256,7 +305,7 @@ Quantimodo = function () {
                     var that = this;
                     jQuery.ajax(this.params.baseURL + url, {
                         beforeSend: function (xhr) {
-                            xhr.setRequestHeader("Authorization", "Bearer " + access_token);
+                            xhr.setRequestHeader('Authorization', 'Bearer ' + access_token);
```

Comment:
Line is too long.<br>Identifier 'access_token' is not in camel case.<br>'access_token' is not defined.